cppSettings.gradle

```diff
@@ -198,6 +235,26 @@ subprojects {
             addStaticWpiUtilLibraryLinks(compileTask, linker, targetPlatform)
         }
     }
+
+    // This defines a project property that projects depending on cscore can use to setup that dependency.
+    ext.defineCsCoreProperties = {
+        ext.csCore = csCoreUnzipLocation
+        ext.csCoreInclude = "$csCoreUnzipLocation/include"
+        ext.csLibArmLocation = "$csCoreUnzipLocation/lib"
+        ext.csSharedLib = "$csLibArmLocation/libcscore.so"
+        ext.cvSharedLib = "$csLibArmLocation/libopencv.so"
+
+        ext.addCsCoreLibraryLinks = { compileTask, linker, targetPlatform ->
+            compileTask.dependsOn project(':').unzipCsCore
+            String architecture = targetPlatform.architecture
+            if (architecture.contains('arm')) {
+                linker.args << '-L' + csLibArmLocation
```

Comment:
If you're adding the location, you don't need to provide the full path to the libraries, you can just say e.g. `-lcscore`.